x-pack/plugins/ml/public/register_helper/register_search_links/search_deep_links.ts

```diff
@@ -34,6 +34,30 @@ const DATA_FRAME_ANALYTICS_DEEP_LINK: AppDeepLink = {
   path: `/${ML_PAGES.DATA_FRAME_ANALYTICS_JOBS_MANAGE}`,
 };
 
+const EXPLAIN_LOG_RATE_SPIKES_DEEP_LINK: AppDeepLink = {
```

Comment:
I think it's worth nesting these inside an extra 'AIOps Labs' level into these links, in a similar way to what we have for the settings pages:

<img width="662" alt="image" src="https://user-images.githubusercontent.com/7405507/233123056-c641c43e-ef02-4aba-ab4e-03c0fb17ace4.png">

Reply:
Added the 'AIOps' level in 1c02d7515fa7fcb808be32922f5448453437057c

cc @walterra